CONTRIBUTING.md

```diff
@@ -8,7 +8,9 @@ Thank you for your interest in contributing to `hypersdk`! By contributing to hy
 
 To contribute to `hypersdk`, you'll need:
 
-- [Go](https://golang.org/dl/) 1.22.8 or higher
+- [Go](https://golang.org/dl/) 1.23.7 or higher
+
+On MacOS, a modern version of bash is required (e.g. via [homebrew](https://brew.sh/) with `brew install bash`). The version installed by default is not compatible with HyperSDK's [shell scripts](scripts).
```

Comment:
Could we mention and link the optional dev shell here as well? 

imo it's preferred to treat this as a convenient option rather than a requirement and specify (as we do here) the installation requirements for using / contributing to HyperSDK.